.ci/magic-modules/pyutils/strutils.py

```diff
@@ -35,4 +37,73 @@ def parse_github_url(gh_url):
   if matches:
     repo, prnum = matches.groups()
     return (repo, int(prnum))
-  return None
\ No newline at end of file
+  return None
+
+def get_release_note(body):
+  """Parse release note block from a given text block.
+
+  Finds the first markdown code block with a "releasenote" language class.
+  Example:
+    ```releasenote
+    This is the release note
+    ```
+
+  Args:
+    body (string): PR body to pull release note block from
+
+  Returns:
+    Release note if found or empty string.
+  """
+  m = re.search(RELEASE_NOTE_RE, body, re.MULTILINE)
+  return m.groupdict("")["release_note"].strip() if m else ""
+
+def set_release_note(release_note, body):
+  """Sanitize and adds the given release note block for PR body text.
+
+  For a given text block, removes any existing "releasenote" markdown code
+  blocks and adds the given release note at the end.
+
+  Example:
+    # Set a release note
+    > print set_release_note(
+        "This is the new release note",
+        "``releasenote\nChanges to downstream\n```\n")
+    "```releasenote\nThis is the new release note\n```\n"
+
+    # Remove for empty release note
+    > print set_release_note("",
+        "PR description\n```releasenote\nChanges to downstream\n```\n")
+    "PR description\n"
+
+  Args:
+    release_note (string): Release note to add
+    body (string): Text body to find and edit release note blocks in
+
+  Returns:
+    Modified text
+  """
+  edited = re.sub(RELEASE_NOTE_RE, '', body)
+  release_note = release_note.strip()
+  if release_note:
+    edited += "\n```releasenote\n%s\n```\n" % release_note.strip()
+  return edited
+
+def find_prefixed_labels(labels, prefix):
+  """Util for filtering and cleaning labels that are changelog-relevant.
+
+  Given a list of labels, find only the changelog-specific labels and
+  removes the prefix indicating they are for downstreams.
```

Comment:
I don't think it removes the prefix

Reply:
i'm adding it back so it does - sorry, wasn't actually asking for review yet :) just wanted you to see my naming change